src/shared/ui/InputFiled/index.tsx

```
@@ -0,0 +1,47 @@
+        return <TextAreaTypeFiled name={name} {...rest} />;
+      case 'image':
+        return <ImageTypeFiled imageProps={imageProps} className={rest.className} />;
+
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <div className={cn('flex h-full w-full flex-col items-start gap-2')}>
+      {label && <label className={cn(S.defaultLabel, labelStyle)}>{label}</label>}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)